stdlib/LinearAlgebra/test/blas.jl

```diff
@@ -200,6 +201,48 @@ Random.seed!(100)
             @test_throws DimensionMismatch BLAS.trmm('R','U','N','N',one(elty),triu(Cnn),Cnm)
         end
 
+        #hpmv!
+        @testset "hpmv!" begin
+            if elty == ComplexF32 || elty == ComplexF64
```

Comment:
Nitpicking, but maybe you should stick a little closer to the style in this test file. The comment line is probably obsolete, and the testset should be set up only if `elty` is complex, similar to `her2k!` in line 26/27:
```julia
if elty in (ComplexF32, ComplexF64)
    @testset "hpmv!" begin
```
Finally, your tests are inside the `let`-block started on line 55/56, within which `n = 10` is used for vector lengths/matrix sizes. You could replace your loop over `N in 2:5` by a single test with `n = 10` instead of `N in 2:5`, or move your tests outside of the block.

Reply:
Moved the ```if``` statement outside of the ```@testset```. Use predefined ```n``` as the matrix dimension.